packages/contentful/migrations/gp2/projects/20230531104312-remove-keywords.js

```diff
@@ -0,0 +1,63 @@
+module.exports.description = 'remove keyword validation';
+
+module.exports.up = (migration) => {
+  const projects = migration.editContentType('projects');
+  projects.editField('keywords').items({
+    type: 'Symbol',
+    validations: [],
+  });
+  projects.changeFieldControl('keywords', 'builtin', 'checkbox', {});
```

Comment:
If you don't have predefined values how can the field control be a checkbox? You probably need it to be an open text input.

Reply:
maybe a tageditor?

  projects.changeFieldControl('keywords', 'builtin', 'tagEditor', {});